lib/runAll.js

```diff
@@ -102,38 +101,48 @@ https://github.com/okonet/lint-staged#using-js-functions-to-customize-linter-com
     return 'No tasks to run.'
   }
 
-  return new Listr(
-    [
-      {
-        title: 'Stashing changes...',
-        skip: async () => {
-          const hasPSF = await git.hasPartiallyStagedFiles({ cwd: gitDir })
-          if (!hasPSF) {
-            return 'No partially staged files found...'
-          }
-          return false
+  const git = new GitWorkflow(gitDir)
+
+  try {
+    await new Listr(
+      [
+        {
+          title: 'Preparing...',
+          task: () => git.stashBackup()
+        },
+        {
+          title: 'Running tasks...',
+          task: () => new Listr(tasks, { ...listrOptions, concurrent: true, exitOnError: false })
         },
-        task: ctx => {
-          ctx.hasStash = true
-          return git.gitStashSave({ cwd: gitDir })
+        {
+          title: 'Applying unstaged changes...',
```

Comment:
Not sure what this means. Can it be rephrased?

Reply:
Unstaged changes are stashed away before running tasks against the staged files. During this step, they will be applied back. Maybe "Restoring unstaged changes..."?

Reply:
Yes, that’s much better!